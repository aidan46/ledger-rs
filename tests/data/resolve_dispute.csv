type,       client, tx, amount
deposit,         2,  2,    2
withdrawal,      2,  5,    3
dispute,         2,  2
resolve,         2,  2
