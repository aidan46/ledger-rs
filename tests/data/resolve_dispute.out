client,available,held,total,locked
2,2,0,2,false
